添加了关于环境变量的内容

jwnhy opened the pull request:
添加了关于`rust-objdump`安装后可能需要的环境变量配置，避免无谓的时间浪费。

jwnhy committed:
添加了关于环境变量的内容

添加了关于`rust-objdump`安装后可能需要的环境变量配置，避免无谓的时间浪费。

Tuyixiang reviewed (changes requested):
感谢补充，确实是一个新手容易遇到的问题。

不过有些小问题还需要处理：

- `$HOME` 需要大写
- 标题的 `rust-objdump` 应当使用代码格式
- English、`code` 等文字和两边汉字之间需要加空格。这个属于在仓库 `docs/format` 下面的一些关于文档排版的要求（不过没有仔细整理所以可能有点乱）

jwnhy committed:
修改格式以符合代码规范

按照如下要修改了格式
    $HOME 需要大写
    标题的 rust-objdump 应当使用代码格式
    English、code 等文字和两边汉字之间需要加空格。这个属于在仓库 docs/format 下面的一些关于文档排版的要求（不过没有仔细整理所以可能有点乱）

jwnhy committed:
无伤大雅的小修正

global少了个a

jwnhy commented:
已经按照要求修改了格式，并且对第六部分加上了一个小修正。

Tuyixiang reviewed (changes requested):
代码中使用的就是 `.globl`，这不是一个 typo

jwnhy committed:
Revert "无伤大雅的小修正"

This reverts commit b5b19a3523ea86fbffde8570c0129da02e25e8f4.

jwnhy commented:
撤回了，能问一下为什么吗？

Tuyixiang commented:
> 撤回了，能问一下为什么吗？

`.globl` 和 `.global` 编译器都支持，使用前者可能是因为历史原因

jwnhy committed:
Merge remote-tracking branch 'upstream/master'
